token/services/db/driver/audit.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package driver
+
+import (
+	view2 "github.com/hyperledger-labs/fabric-smart-client/platform/view"
+	token2 "github.com/hyperledger-labs/fabric-token-sdk/token"
+)
+
+// AuditTransactionDB defines the interface for a token transactions database
```

Comment:
Maybe we could have a more complete description of what these DB's are used for.

Reply:
👍